src/test/java/Categorytest.java

```diff
@@ -0,0 +1,78 @@
+import Base.BaseClass;
+import Page.Categories;
+import Page.Login;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+public class Categorytest {
+
+    WebDriver driver;
+    @BeforeTest
+    void login() throws InterruptedException {
+        driver = BaseClass.init();
+        Login.login(driver,"vivek", "vivek123");
+
+    }
+    @Test(priority = 1)
+    void categoriestest(){
+        Categories.category(driver);
+        Assert.assertEquals(Categories.actualtext(driver),Categories.expectedtext(driver));
+        System.out.println("Assertion Passed");
+
+    }
+    @Test(priority = 2)
+    void fillvaluetest() throws IOException {
+        Categories.category(driver);
+        File file = new File("src/test/resources/categoryReqData.xlsx");
```

Comment:
Put file options in a generic function.So that we can keep one copy and whenever required will call that function.